tendrl/node_agent/flows/generate_journal_mapping/__init__.py

```diff
@@ -0,0 +1,16 @@
+import json
+
+from tendrl.commons import flows
+from tendrl.node_agent.provisioner.ceph import utils
+
+
+class GenerateJournalMapping(flows.BaseFlow):
+    def run(self):
+        # Generate the journal mapping for the nodes
+        mapping = utils.generate_journal_mapping(
+            self.parameters['Cluster.node_configuration']
+        )
+        NS.etcd_orm.client.write(
+            "/queue/%s/output" % self.job_id,
```

Comment:
Sent PR https://github.com/Tendrl/commons/pull/410 for setting default empty dict for `output` field of jobs.